Sources/WalletConnectPush/Push.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import WalletConnectNetworking
+import WalletConnectPairing
+import WalletConnectEcho
+
+public class Push {
+
+    class Dapp {
+        public static var instance: DappPushClient = {
```

Comment:
What you think about: Push.dapp, Push.wallet, instead of Push.Dapp.instance and Push.Wallet.instance ?

Reply:
hmm maybe not a bad idea

Reply:
applied